readme.md

```diff
@@ -187,6 +206,10 @@ WP Local Docker is an ever-evolving tool, and it's important to keep your local
 
 It's especially important to `git pull` the latest code before you `docker pull` upgrades to your Docker images, either as a potential fix for an issue or just to make sure they’re running the latest versions of everything. This will make sure you have the latest WP Local Docker code first, including the `docker-compose.yml` file that defines what Docker images and versions the environment uses.
 
+## Support Level
+
+**Active:** 10up is actively working on this, and we expect to continue work for the foreseeable future including keeping tested up to the most recent version of WordPress.  Bug reports, feature requests, questions, and pull requests are welcome.
```

Comment:
@jeffpaul Lets put stable for this repo (v2 should be active)

Reply:
done, see: https://github.com/10up/wp-local-docker/pull/142/commits/3b7a59f635021fa1a00886477d2edd93ddbd35c3